src/Tools/TransformXMLToCSVTool/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Reflection;
+using System.IO;
+using System.Text;
+using System.Configuration;
+using System.Collections.Generic;
+using System.Xml.Xsl;
+using System.Linq;
+using System.Collections.Specialized;
+
+namespace TransformXMLToJSONUsingXSLT
+{
+    class Program
+    {
+        static string DirectoryPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().GetModules()[0].FullyQualifiedName);
+        static string xmlFilePathResult = ConfigurationManager.AppSettings["xmlFilePathResult"];
+        static string xsltFile = ConfigurationManager.AppSettings["xsltFile"];
+        static string textFileComplete = ConfigurationManager.AppSettings["textFileComplete"];
+        static string textFileWithFiltersSlowTests = ConfigurationManager.AppSettings["textFileWithFiltersSlowTests"];
+        static string textFileWithFiltersFastTests = ConfigurationManager.AppSettings["textFileWithFiltersFastTests"];
+        static List<String> allLines = new List<string>();
+        static List<String> restLines = new List<string>();
+
+        static void Main(string[] args)
+        {
+            XSLTToAnyType();
+        }
+        public static void XSLTToAnyType()
```

Comment:
@manuelsaldivar525 looking sold- please add ///summary tags to public methods explaining what the method does.

Reply:
Yes, I added documentation.